packages/combobox/src/Combobox.styles.ts

```diff
@@ -1,207 +1,165 @@
-/**
- * Styles
- */
-
-import { css, cx, keyframes } from '@leafygreen-ui/emotion';
+import { css, cx } from '@leafygreen-ui/emotion';
 import { createUniqueClassName } from '@leafygreen-ui/lib';
 import { uiColors } from '@leafygreen-ui/palette';
-import { fontFamilies, typeScales } from '@leafygreen-ui/tokens';
-import { isArray } from 'lodash';
-import { ComboboxSize, Overflow, State } from './Combobox.types';
+import isNull from 'lodash/isNull';
+import { fontFamilies, spacing, typeScales } from '@leafygreen-ui/tokens';
+import { transparentize } from 'polished';
+import { ComboboxSize as Size, Overflow, Theme } from './Combobox.types';
 
 /**
  * Width of the widest character (in px)
  */
-const maxCharWidth: Record<ComboboxSize, number> = {
-  [ComboboxSize.Default]: typeScales.body1.fontSize,
-  [ComboboxSize.Large]: typeScales.body2.fontSize,
+export const maxCharWidth: Record<Size, number> = {
+  [Size.Default]: typeScales.body1.fontSize,
+  [Size.Large]: typeScales.body2.fontSize,
 };
 
-export const chipClassName = createUniqueClassName('combobox-chip');
-
-// TODO: Remove this during refresh update
-export const _tempLabelDescriptionOverrideStyle = css`
-  font-family: ${fontFamilies.legacy};
-  font-size: var(--lg-combobox-font-size);
-  line-height: var(--lg-combobox-line-height);
-`;
+/**
+ * Vertical padding on a chip (in px)
+ */
+export const chipWrapperPaddingY = {
+  [Size.Default]: 2,
+  [Size.Large]: 4,
+} as const;
 
-export const comboboxParentStyle = ({
-  darkMode,
-  size,
-  overflow,
-}: {
-  darkMode: boolean;
-  size: ComboboxSize;
-  overflow: Overflow;
-}) => {
-  const modeStyle = (darkMode: boolean) => {
-    if (darkMode) {
-      return css``;
-    } else {
-      return css`
-        --lg-combobox-color-error: ${uiColors.red.base};
-        --lg-combobox-text-color: ${uiColors.gray.dark3};
-        --lg-combobox-text-color-disabled: ${uiColors.gray.dark1};
+/**
+ * Height of the input element (in px)
+ */
+const inputHeight: Record<Size, number> = {
+  [Size.Default]:
+    typeScales.body1.lineHeight + 2 * chipWrapperPaddingY[Size.Default], // 20
+  [Size.Large]:
+    typeScales.body2.lineHeight + 2 * chipWrapperPaddingY[Size.Large], // 28
+};
 
-        --lg-combobox-background-color: ${uiColors.gray.light3};
-        --lg-combobox-background-color-focus: ${uiColors.white};
-        --lg-combobox-background-color-disabled: ${uiColors.gray.light2};
+/**
+ * Size of combobox x & y padding (in px)
+ */
+export const comboboxPadding: Record<Size, { x: number; y: number }> = {
+  [Size.Default]: {
+    y: (36 - inputHeight[Size.Default] - 2) / 2,
+    x: 8 - 1,
```

Comment:
Curious, where are `8` and `1` coming from and why can't you use `7`?

Reply:
I initially had `spacing[2]`, but decided to hard code it. Adding the `-1` to make it clear that it's the border. I'll change it back to use `spacing`